scripts/auto-update-and-commit.sh

```diff
@@ -11,5 +11,6 @@ git branch -D latest || true
 git checkout -b latest
 
 git add submodules
-git commit -m "Auto: Update versions to latest"
+git commit -m "Auto: Update versions to latest
+$message"
```

Comment:
Does this leave a one line gap between the title and the body? It looks like one more newline might be needed between lines 14 & 15. Without the blank line, the "body" part won't render as expected in some tools